dts/bindings/sensor/adi,adxl372-common.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2022 Analog Devices Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+properties:
+    odr:
+      type: int
+      required: false
+      default: 0
```

Comment:
Default property values need justification (e.g., is it the power on reset value?) in the description. See here for more details: https://docs.zephyrproject.org/latest/build/dts/bindings.html#rules-for-default-values